repos/spack_repo/builtin/packages/chapel/package.py

```diff
@@ -196,6 +197,14 @@ class Chapel(AutotoolsPackage, CudaPackage, ROCmPackage):
         values=("gasnet", "none", "ofi", "ugni"),
     )
 
+    variant(
+        "comm_ofi_oob",
+        values=("sockets", "mpi", "pmi2", "unset"),
+        default="unset",
+        description="Select out-of-band support",
```

Comment:
Suggest expanding this description to make it easier for users to find the relevant Chapel documentation:

```suggestion
        description="Select out-of-band support (CHPL_COMM_OFI_OOB)",
```